internal/build/build.go

```diff
@@ -194,7 +194,7 @@ func checkBinarySizeActionFunc(c *cli.Context) (err error) {
 		helloSourceFilePath  = "./internal/example-hello-world/example-hello-world.go"
 		helloBuiltFilePath   = "./internal/example-hello-world/built-example"
 		desiredMinBinarySize = 2.0
-		desiredMaxBinarySize = 2.1
+		desiredMaxBinarySize = 2.2
```

Comment:
The binary size increasing and the new dependency (`golang.org/x/crypto`) concern me, can you rewrite this PR to remove those two changes?

Reply:
I can experiment with some platform-specific code, and see what is possible. Would it be OK if this feature only works on linux/mac/windows (for a small subset of each that I can easily test)?

Reply:
I needs to work on all of the platforms that we test on CI, and you don't need to support anything beyond that

Reply:
Even using the low-level `unix.IoctlGetWinsize`/`windows.GetConsoleScreenBufferInfo` calls, this increases the binary size by 0.1M. Though perhaps golang.org/x/sys/unix and golang.org/x/sys/windows are more likely to already be used by client programs than golang.org/x/crypto.

The only other option I can think of to not raise the binary size by this much would be for clients to figure out the terminal size themselves and provide it somehow (maybe set it as a new field in the App struct). What do you think?

Reply:
There's 2 options here that stand out for me:

1. import the code for the `unix.IoctlGetWinsize` and `windows.GetConsoleScreenBufferInfo` functions so that we aren't importing the whole package
2. allow people to manually set the terminal size, like you mention

Either way, we definitely want to avoid increasing the binary size for this feature. At least, not in a pre-v3 world where can't make this feature opt-in. (see https://github.com/urfave/cli/issues/833#issuecomment-579869829 for context)

Reply:
I would also say that it would probably be un-controversial to let people set the terminal size, and have a default terminal size of `80` as described here => https://github.com/urfave/cli/issues/287#issuecomment-320581746

Reply:
I would rather not copy+paste all that subtle syscall code, so I think we should go for (2) - it also makes this feature testable.